shared/tinyfiledialogs/Makefile

```diff
@@ -0,0 +1,2 @@
+HEADERS += $(SHARED_SRC_DIR)/tinyfiledialogs/tinyfiledialogs.h
+SOURCES += $(SHARED_SRC_DIR)/tinyfiledialogs/tinyfiledialogs.cpp
```

Comment:
This will cause the file to be built relative to `.eobjs` as `shared/` is relative to `SHELL/`.
You should be adding this to SHARED_SOURCES and handling those sources in the engine Makefile with the three lines I gave you earlier.

Reply:
You mind sending those 3 lines again? It's really buried on on discord and I don't feel like looking for it.

Reply:
I honestly don't know what you want me to do @JoshDreamland

Reply:
You're not being specific enough as usual.